### DISCUSSION TITLE
error during installing clang8 on Ubuntu 20.4

### DISCUSSION THREAD

**Discussion by iscanGH** [actions/virtual-environments#4927] [Installed software]
I need to install clang-8 to be compatible with another package (Carla). 
First, I removed clang-11 and its dependencies (I think), and 
`sudo apt-get update`

Then I failed at 
`sudo apt-get install clang-8`
`The following packages have unmet dependencies:
 clang-8 : Depends: libclang-common-8-dev (= 1:8.0.1+svn369350-1~exp1~20200114191400.80) but 1:8.0.1-9 is to be installed
E: Unable to correct problems, you have held broken packages.
`
I installed aptitude and also run several combinations of resolving issues, but failed. For example when I run 
`sudo aptitude install clang-8`
I get
`The following NEW packages will be installed:
  gcc-7-base{a} lib32gcc1{a} libasan4{a} libcilkrts5{a} libgcc-7-dev{a} 
  libobjc-7-dev{a} libstdc++-7-dev{a} libubsan0{a} 
The following packages will be REMOVED:
  libobjc-8-dev{u} 
The following packages will be upgraded:
  clang-8 libclang-common-8-dev libclang1-8 libllvm8{b} 
4 packages upgraded, 8 newly installed, 1 to remove and 3 not to upgrade.
Need to get 36.2 MB of archives. After unpacking 35.8 MB will be used.
The following packages have unmet dependencies:
 libllvm8 : Depends: libffi6 (>= 3.0.4) which is a virtual package and is not provided by any available package

 llvm-8-dev : Depends: libllvm8 (= 1:8.0.1-9) but 1:8.0.1+svn369350-1~exp1~20200114191400.80 is to be installed
The following actions will resolve these dependencies:

     Remove the following packages:                       
1)     clang-8 [1:8.0.1-9 (focal, now)]                   

     Keep the following packages at their current version:
2)     libclang-common-8-dev [1:8.0.1-9 (focal, now)]     
3)     libllvm8 [1:8.0.1-9 (focal, now)] 

Accept this solution? [Y/n/q/?] 
`

**Comment by vvnchrsto** [comment]
I have the exact same issue. If anyone can help, it would be highly appreciated.

**Comment by miketimofeev** [comment]
@vvnchrsto take a look at this issue, please https://github.com/actions/virtual-environments/issues/4930
